src/test/java/org/broadinstitute/consent/http/resources/MetricsResourceTest.java

```diff
@@ -32,21 +30,22 @@ private void initResource() {
 
   @Test
   public void testGetDarMetricsData() {
-    when(service.generateDarDecisionMetrics()).thenReturn(Collections.emptyList());
+    when(service.generateDecisionMetrics(Type.DAR)).thenReturn(Collections.emptyList());
     initResource();
     Response response = resource.getDarMetricsData();
     assertEquals(200, response.getStatus());
     assertFalse(response.getEntity().toString().isEmpty());
-    assertTrue(response.getEntity().toString().contains(DarDecisionMetrics.getHeaderRow("\t")));
+    assertTrue(response.getEntity().toString().contains(service.getHeaderRow(Type.DAR)));
   }
 
   @Test
   public void testGetDacMetricsData() {
-    when(service.generateDarDecisionMetrics()).thenReturn(Collections.emptyList());
+    when(service.generateDecisionMetrics(Type.DAC)).thenReturn(Collections.emptyList());
```

Comment:
no need to change this for the PR, but you can also use `any()` in place of `Type.DAC` which would work in the case of either enum type.